samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -10,43 +10,52 @@ class TeamsMessagingExtensionsActionBot extends TeamsActivityHandler {
                 return createCardCommand(context, action);
             case 'shareMessage':
                 return shareMessageCommand(context, action);
+            case 'webView':
+                return await webViewResponse(context, action);
             default:
                 throw new Error('NotImplemented');
         }
     }
 
     async handleTeamsMessagingExtensionFetchTask(context, action) {
-        try {
-           const member = await this.getSingleMember(context);
-           return {
-                task: {
-                    type: 'continue',
-                    value: {
-                        card: GetAdaptiveCardAttachment(),
-                        height: 400,
-                        title: 'Hello ' + member,
-                        width: 300
-                    },
-                },
-            };
-        } catch (e) {
-            if (e.code === 'BotNotInConversationRoster') {
+        switch (action.commandId) {
+            case 'webView':
+               return empDetails(context, action);
+            case 'Static HTML':
+               return dateTimeInfo(context, action);
+            default:
+               try {
+                const member = await this.getSingleMember(context);
```

Comment:
Spacing. I believe `const member` and the `return` below it need to be indented some.